python/pysrc/kaskada/_timestream.py

```diff
@@ -880,6 +893,84 @@ def record(self, fields: Callable[[Timestream], Mapping[str, Arg]]) -> Timestrea
         """
         return record(fields(self))
 
+    def round(self) -> Timestream:
+        """Return a Timestream with all values rounded to the nearest integer.
+
+        Returns:
+            A Timestream of the same type as `self`. The result contains `null`
```

Comment:
I've been using "at that point" instead of "position", trying to line the terms up more with timelines/timestreams.